{{cookiecutter.repo_name}}/pyproject.toml

```diff
@@ -133,6 +135,8 @@ pylint = ["-*"]
     [tool.pylint.format]
     max-line-length = "139"
 
+[tool.cruft]
+skip = [".git", "node_modules"]
```

Comment:
Works around https://github.com/cruft/cruft/issues/61

I'm less sure why `node_modules` needs to be here as it's not in the template, but without it it'll try to compare files in there and choke.